tutorials/matchmaker101/matchmaker.yaml

```diff
@@ -112,3 +112,25 @@ spec:
     protocol: TCP
     port: 50508
 ---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: om-configmap-override
+  namespace: open-match
+  annotations:
+    chart: open-match-0.0.0-dev
+    heritage: Helm
```

Comment:
could you remove the `annotations` part and `labels.release` from the configmap? These fields are popped up by helm